koin/src/main/java/in/koreatech/koin/ui/land/NaverMapFragment.kt

```diff
@@ -0,0 +1,35 @@
+package `in`.koreatech.koin.ui.land
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.MotionEvent
+import android.view.View
+import android.view.ViewGroup
+import com.naver.maps.map.MapFragment
+import com.naver.maps.map.NaverMapOptions
+
+class NaverMapFragment: MapFragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = super.onCreateView(inflater, container, savedInstanceState)
+        view?.setOnTouchListener { touchView: View, event: MotionEvent? ->
+            touchView.parent.requestDisallowInterceptTouchEvent(true)
+            mapView!!.onTouchEvent(event)
```

Comment:
```suggestion
            mapView?.onTouchEvent(event)
```
으로 변경해주세요. `!!` 는 가급적 삼가해주세요.